web/src/app/globals.css

```diff
@@ -1,6 +1,8 @@
 :root {
   --background: #ffffff;
   --foreground: #171717;
+
+  --cta-green: #165823;
```

Comment:
Might be worth making this more specific. For instance, does this refer to the colour of the text or something else (e.g. border)?